doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,128 @@
+
+The nrf70 series device supports raw packet transmission in the below station operating modes:
+
+* **Station idle mode:** The device is operating as an 802.11 station device and is in idle state (not connected to AP).
+* **Station connected mode:** The device is operating as an 802.11 station device and is in connected state.
+
+To enable raw packet transmission, the nrf70 device mode of operation has to be configured to "Station + TX-Injector mode" using the ``NET_REQUEST_WIFI_MODE`` net_mgmt API. When the raw tx mode is enabled, a channel for raw packet transmission also has to be configured mandatorily. This channel will be utilized for raw packet transmission when the station is in disconnected state. When the station is in connected state, the station connected channel will be used for raw packet transmission. Channel configuration cannot be done when station is in connected state. The net_mgmt API ``NET_REQUEST_WIFI_CHANNEL`` can be used to configure the channel of operation.
+
+To disable raw packet transmission, the nrf70 device mode of operation has to be configured to "**station mode**" using the ``NET_REQUEST_WIFI_MODE`` net_mgmt API.
+
+See the :ref:`wifi_shell_sample` sample for more information on configuring mode setting and channel setting for raw packet transmission via shell commands.
+
+The shell commands and network management APIs to swtich between operational modes and to set channel are provided below
+
+.. list-table:: Wi-Fi raw packet transmission network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Wi-Fi shell command
+     - Example Usage
+     - Description
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -st
+     - configure interface instance 1 to Station + TX injector mode
+   * - net_mgmt(NET_REQUEST_WIFI_MODE)
+     - wifi mode -i<interface instance> <configuration>
+     - wifi mode -i1 -s
+     - configure interface instance 1 to pure station mode
+   * - net_mgmt(NET_REQUEST_WIFI_CHANNEL)
+     - wifi channel -i<interface instance> <channel number to set>
+     - wifi channel -i1 -c6
+     - set raw transmission channel to 6 for interface 1 to be used in Station idle mode
+
+Raw packet transmit header
+**************************
+
+The raw packet to be transmitted by an application should be a fully conformant 802.11 packet. In addtion, it should provide transmit parameters that inform the nrf70 series Wi-Fi device driver on how the packet should be transmitted. It should also provide a packet marker identifying itself as a special packet which should be handled differently by the nrf70 series Wi-Fi driver. The transmit parameters and the packet marker together forms the raw packet transmit header. The raw packet transmit header is prepended to the 802.11 conformant raw packet before transmitting the packet to the lower layers by the application.
```

Comment:
Suggest to have a sub-title, like: "Packet format rules"

We need a brief description of what we mean by fully conformant IEEE 802.11 packet"
My understanding after talking to @VivekUppunda  is that we mean that
- the 802.11 MAC header is present and valid
- the FCS field is present and valid
Anything else?

Also this is not a "should", but a "must", since, according to my understanding the nRF70 Series will drop frames that do not appear as fully conformant. E.g.

"The application using the raw packet API must construct and provide fully-conformant....."

Reply:
Transmit & PHY  parameters could be a second sub-title here and you can explain what happens :)

Reply:
added transmit and PHY parameters for raw packet header elements